app/editor/src/components/form/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { SelectHTMLAttributes } from 'react';
+
+import { DropdownVariant, instanceOfIOption, IOption } from '.';
+import * as styled from './DropdownStyled';
+
+export interface IDropdownProps extends SelectHTMLAttributes<HTMLSelectElement> {
+  /**
+   * The control name.
+   */
+  name: string;
```

Comment:
Moving `name` and `label` from the formik version to the base version.  Need to do this for the other components too.